test/src/kbasesearchengine/test/integration/IndexerWorkerIntegrationTest.java

```diff
@@ -12,6 +12,8 @@
 import java.util.Set;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
```

Comment:
My point was to make sure unit tests pass before even trying to write integration tests. I hadn't looked at the code at that point so I wasn't trying to say 'you must write unit tests and then write integration tests', because in most cases integration tests are overkill. I was objecting to trying to debug integration tests before passing unit tests existed.